cmd/mount.go

```diff
@@ -100,7 +100,7 @@ be interacting with the file system.`)
 		OpRateLimitHz:                      newConfig.GcsConnection.LimitOpsPerSec,
 		StatCacheMaxSizeMB:                 statCacheMaxSizeMB,
 		StatCacheTTL:                       metadataCacheTTL,
-		EnableMonitoring:                   newConfig.Metrics.StackdriverExportInterval > 0,
+		EnableMonitoring:                   newConfig.Metrics.StackdriverExportInterval > 0 || newConfig.Metrics.PrometheusPort != 0,
```

Comment:
Hi @sethiay , @kislaykishore I added this line, and then from tests I can see `gcs_reader_count`, `gcs_request_count`, and `gcs_request_latencies` metrics. However, I still don't see `gcs_read_bytes_count` metric. Can you help take a look and see if I am missing anything?

Reply:
There is a bug in how this counter is computed - it's not updated unless it's a slightly large file and then too it's updated incorrectly. I'm working on https://github.com/GoogleCloudPlatform/gcsfuse/pull/2167 to fix this.
For your testing, create a 1G file in GCS and used `cat <filename> > /dev/null` to read it. That should move the counter:
![image](https://github.com/user-attachments/assets/e969da6b-46b2-416a-b3fe-77cfd629a421).

Reply:
Thank you for the fix!